src/Rules/StarterWeb/AI/IndividualAuth/project.json

```diff
@@ -58,6 +53,13 @@
       "version": "__AspNetCorePlatformVersion__",
       "imports": "portable-net45+win8+dnxcore50"
     },
+    "Microsoft.EntityFrameworkCore.Tools": {
+      "version": "__AspNetCorePlatformVersion__",
+      "imports": [
+        "portable-net45+win8+dnxcore50",
+        "portable-net45+win8"
```

Comment:
I haven't tried these imports, so it would be good to verify they works. The import I've been telling people to use is `portable-net451+win8`. Likewise for the imports on netcoreapp1.0